tests/include/ApiTest.php

```diff
@@ -1293,9 +1297,11 @@ public function testApiStatusShowWithXml()
 	 * Test the api_status_show() function with a raw result.
 	 * @return void
 	 */
-	public function testApiStatusShowWithRaw()
+	public function testApiGetLastStatus()
 	{
-		$this->assertStatus(api_status_show('raw'));
+		$user_info = api_get_user($this->app);
```

Comment:
Please don't use two API calls in one test, see below why it's not necessary :-)